docs/web3-eth.rst

```diff
@@ -716,6 +716,43 @@ Example
 
 ------------------------------------------------------------------------------
 
+.. _eth-feehistory:
+
+
+getFeeHistory
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.getFeeHistory(blockCount, newestBlock, rewardPercentiles, [callback])
+
+Transaction fee history
+Returns base fee per gas and transaction effective priority fee per gas history for the requested block range if available. 
+The range between headBlock-4 and headBlock is guaranteed to be available while retrieving data from the pending block and older
+history are optional to support. For pre-EIP-1559 blocks the gas prices are returned as rewards and zeroes are returned for the base fee per gas.
+
+----------
+Parameters
+----------
+
+1. ``String`` - Number of blocks in the requested range. Between 1 and 1024 blocks can be requested in a single query. Less than requested may be returned if not all blocks are available.
```

Comment:
```suggestion
1. ``String|Number|BN|BigNumber`` - Number of blocks in the requested range. Between 1 and 1024 blocks can be requested in a single query. Less than requested may be returned if not all blocks are available.
```

Reply:
Shouldn't `blockCount` just be a `number`? The method takes an int, https://github.com/ethereum/go-ethereum/blob/c49e065fea78a5d3759f7853a608494913e5824e/internal/ethapi/api.go#L90

I guess it depends on if an inputFormatter is applied to force it to format it to a number?

Reply:
Unfortunately its a design decision we need to stick with based on the history of web3.js, the 4.x branch improves on this a bit.

For reference check out [eth_gettransactionfromblock](https://web3js.readthedocs.io/en/v1.4.0/web3-eth.html#gettransactionfromblock)